v3.0.0/tasks-schema.json

```diff
@@ -0,0 +1,1449 @@
+                                                    },
+                                                    "required": [
+                                                        "is_required",
+                                                        "type",
+                                                        "min_samples_per_task",
+                                                        "max_samples_per_task"
+                                                    ],
+                                                    "if": {
+                                                        "properties": {
+                                                            "type": { "const": "character" }
+                                                        }
+                                                    },
+                                                    "then": {
+                                                        "required": ["max_length"]
+                                                    }
+                                                },
+                                                "value": {
+                                                    "type": "object",
+                                                    "description": "Object defining valid values of samples from the predictive distribution for a given sample index.  Depending on the Hub specification, samples with the same sample index (specified by the output_type_id) may be assumed to correspond to a joint distribution across multiple levels of the task id variables. See Hub documentation for details.",
+                                                    "properties": {
+                                                        "type": {
+                                                            "description": "Data type of sample value from the predictive distribution.",
+                                                            "examples": [{
+                                                                "type": "double"
+                                                            }],
+                                                            "type": "string",
+                                                            "enum": [
+                                                                "double",
+                                                                "integer"
+                                                            ]
+                                                        },
+                                                        "minimum": {
+                                                            "description": "The minimum inclusive valid sample value from the predictive distribution.",
+                                                            "type": [
+                                                                "number",
+                                                                "integer"
```

Comment:
question: does this mean any kind of number is acceptable, or that it must be an integer?  while in practice I expect people to enter integers, there's no reason to require this.  and I noticed that in an example in the related hubAdmin PR, `0L` and `1L` were specified.  maybe we could just say the type here should match the type of the value column, allowing for double min and max values to be specified if the expected submission type is double?

Reply:
Indeed what this means is that any number is acceptable and I agree that I see no reason to enforce only integers, in fact that could come back to bite us one day. 

I'm not sure why using integers in the `hubAdmin` examples seems fiddly given integers are an accepted data type for the field but indeed  `0` and `1`s would also work just fine too. Let me know if you want me to change them.  

We can indeed add checks that enforce that the `minimum` and `maximum` fields are the same data type as the `value.type` field but I'm not sure what the benefit of it would be given it doesn't really affect functionality anywhere. e.g. if an admin specified `integer` as `type` but `minimum` as 0.5, range validation would still work as expected to ensure all values are above 0.5 while data type validation would ensure all values where integers. So feels like more effort than it's worth.

Reply:
Thanks -- what I meant was, it would seem fiddly if people were forced to enter an integer data type here.  I think this is good as is, thanks for the confirmation!